### DISCUSSION TITLE
On the blog in dex and search results should we show the parent and child relationship between terms in the UI

### DISCUSSION THREAD

**Discussion by SophieAmnesty** [amnestywebsite/humanity-theme#63] [Enhancement Requests]
I feel like the topics list makes more sense and is easier to browse through when you can see that some of the terms are umbrella terms that have sub-categories inside. 

e.g. if I'm interested in content about increased crackdowns on protests. Maybe I scroll down as far as 'censorship and freedom of expression' and select that term. I would have to scroll down to P to see that there's a specific 'protests and demonstrations' term. 

or, I'm interested in discrimination against women. do I click on 'women's rights' or 'women and girls'. It would be easier to know which term to select if I could see that 'women's rights' is a child term of 'discrimination' and 'women and girls' is a parent term for 'domestic violence' and 'maternal health' etc.

**Comment by AmnestyAM** [comment]
I like this idea! Maybe some nesting / indentation in the list / UI? How about regions and countries?

**Comment by SophieAmnesty** [comment]
Yep, I think nesting / indentation would work! 

I'm not sure it's necessary with regions/countries as I think in that case people will have a better idea of what they're looking for and our region categories might be confusing... (e.g. I might start looking underneath Africa for Egypt and then realize it was actually listed under MENA). 

Open to other opinions on this though! :D

**Comment by SophieAmnesty** [comment]
p.s. on second thoughts I think you might be right about including the nesting for the location list as well @AmnestyAM .

**Comment by AmnestyAM** [comment]
Requested by Amnesty SK
